packages/react/src/primitives/PhoneNumberField/CountryCodeSelect.tsx

```diff
@@ -6,30 +6,32 @@ import { ComponentClassNames } from '../shared/constants';
 import { CountryCodeSelectProps, Primitive } from '../types';
 import { SelectField } from '../SelectField';
 
-const CountryCodeSelectPrimitive: Primitive<CountryCodeSelectProps, 'select'> =
-  ({ className, ...props }, ref) => {
-    const countryCodeOptions = React.useMemo(
-      () =>
-        countryDialCodes.map((dialCode) => (
-          <option key={dialCode} value={dialCode}>
-            {dialCode}
-          </option>
-        )),
-      [countryDialCodes]
```

Comment:
Since `countryDialCodes` is a contant, there's no need to list it as a dependency.